src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/wrapped/WrappedStaticStateRequestManager.java

```diff
@@ -17,16 +17,8 @@ public WrappedStaticStateRequestManager(@NotNull final IStandardRequestManager w
         super(wrappedManager);
     }
 
-    @NotNull
     @Override
-    /**
-     * Method to update the state of a given request.
-     *
-     * @param token The token that represents a given request to update.
-     * @param state The new state of that request.
-     * @throws IllegalArgumentException when the token is unknown to this manager.
-     */
-    public void updateRequestState(@NotNull final IToken token, @NotNull final RequestState state) throws IllegalArgumentException
+    public void updateRequestState(@NotNull final IToken<?> token, @NotNull final RequestState state)
     {
         //TODO: implement when link is created with workers
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1135)